ansible/testbed-cli.sh

```diff
@@ -596,6 +599,22 @@ function cleanup_vmhost
       --vault-password-file="${passwd}" -l "${server}" $@
 }
 
+function install_image
+{
+  testbed_name=$1
+  inventory=$2
+  image_url=$3
+  shift
+  shift
+  shift
+
+  echo "Upgrading image on '$testbed_name'"
+
+  ansible-playbook upgrade_sonic.yml -i "$inventory" -e testbed_name="$testbed_name" -e testbed_file=$tbfile -e upgrade_type=sonic -e image_url="$image_url"
```

Comment:
I like to use ${..} for shell expansion instead of "$var" because with the former, expansion of the variable occurs before the command is called. This takes care of spaces or other special characters for us. This is, however entirely semantics.

Reply:
I believe `${var}` and `$var` both expand at the same time and behave the same either way; it does not take care of spaces and special characters. The difference is that the braces put boundaries around the variable names, so that you can have `sample${var}text` (whereas with `sample$vartext`, `$vartext` would be evaluated as a variable).

Reply:
I'll go ahead and resolve this for now then, it looks like we use the `$var` convention elsewhere in this script which is what I referenced originally.